### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by sv-dubov**

### CODE REVIEW

**Review by nzrii-kysel — APPROVED** [review]
gj👍

File: src/inverseRobot.js [hunk 1]
```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inversed = {};
+
+  for (const key in robot) {
+    if (robot[key] in inversed) {
```
Comment (nzrii-kysel) [inline src/inverseRobot.js:13]: I'd use a safer way to check object properties

File: src/inverseRobot.js [hunk 2]
```diff
@@ -7,7 +7,16 @@
...
+      return null;
+    }
+    inversed[robot[key]] = key;
```
Comment (Denchepornyuk) [inline src/inverseRobot.js:16]:
```suggestion
    if (robot[key] in inversed) {
      return null;
    }

    inversed[robot[key]] = key;
```

Comment (Denchepornyuk) [inline src/inverseRobot.js:16]: create variable for `robot[key]`, repeating code should be in the variables

**Review by Denchepornyuk — CHANGES_REQUESTED** [review]

**Review by Denchepornyuk — APPROVED** [review]

File: src/inverseRobot.js [hunk 3]
```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inversed = {};
+
+  for (const key in robot) {
+    const robotKey = robot[key];
+
+    if (robotKey in inversed) {
```
Comment (Denchepornyuk) [inline src/inverseRobot.js:15]: it's better to use hasOwnProperty instead of `in`

Comment (sv-dubov) [reply src/inverseRobot.js]: Done!

**Pull request closed without merging**